changelog/1292.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
```

Comment:
Changelogs aren't used for non-code changes; do you mind removing this?

Thanks!